Bottom: 757eeac7c635416b3f9bc9d33daf67f5f1e847ae
Top:    95ac06b46399ab2e7ae0031bd8574e481698c472
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-05 12:55:56 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!










---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index d7033e1..1237607 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -323,13 +323,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
